fjord/analytics/forms.py

```diff
@@ -46,3 +46,13 @@ class Meta:
             'on_picker',
             'notes'
         ]
+
+class LeftDashboard(forms.Form):
```

Comment:
This is not a good name, because "Left" has nothing to do with what these controls are, and it isn't a dashboard. Maybe something like "DashboardFilterControls"?